README.md

```diff
@@ -1,11 +1,18 @@
 # Find a Doc - Localization
 
-Find a Doc is a free online resource aimed at helping connect the foreign community in Japan with health services in their native language.
+[Find a Doc](https://findadocjp.org/) is a free online resource aimed at helping connect the foreign community in Japan with health services in their native language.
```

Comment:
Update to findadoc.jp

Reply:
Can you add it and merge on my behalf? Otherwise I can get to it late tonight